go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/nats-io/nats.go v1.25.0 // indirect
 	github.com/nats-io/nkeys v0.4.4 // indirect
 	github.com/nats-io/nuid v1.0.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
hmm, what is bringing in this? Run a `go mod tidy` and make sure it still belongs.

Reply:
it's used in credentials.go

Reply:
@brianointel No, that uses `github.com/edgexfoundry/go-mod-core-contracts/v2/errors`, and if you check that package, it is not listed as a dependency either. https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/go.mod.